update.php

```diff
@@ -431,3 +431,29 @@ function update_1347()
 
 	return Update::SUCCESS;
 }
+
+function pre_update_1348()
+{
+	DBA::insert('contact', ['nurl' => '']);
+	DBA::update('contact', ['id' => 0], ['id' => DBA::lastInsertId()]);
+
+	// The tables "permissionset" and "tag" could or could not exist during the update.
+	// This depends upon the previous version. Depending upon this situation we have to add
+	// the "0" values before adding the foreign keys - or after would be sufficient.
+
+	update_1348();
```

Comment:
Wouldn't the `update_1348` function be called anyway? Won't it be ran twice because of this explicit call?

Reply:
One the calls will fail.

But you are right, we need to check whether we inserted something.